tools/mo/openvino/tools/mo/utils/version.py

```diff
@@ -17,9 +21,10 @@ def generate_mo_version():
     custom_{branch_name}_{commit_hash}
     """
     try:
-        branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip().decode()
-        commit_hash = subprocess.check_output(["git", "rev-parse", "HEAD"]).strip().decode()
-        return "custom_{}_{}".format(branch_name, commit_hash)
+        mo_dir = get_mo_root_dir()
+        branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"], cwd=mo_dir).strip().decode()
+        commit_hash = subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=mo_dir).strip().decode()
```

Comment:
You seem not to do a fix for fatal git error. How to fix it?

Reply:
Could you please show an example where you get git fatal error? 

Previous fata errors happened then there was no git repositories in the current folder. But not `git` is run not in the current directory but in the directory where MO is placed and only if version.txt is not found.

Reply:
Communicated personally. The current changed fix error 'fatal git repository not found'